packages/contracts/contracts/registry/DAORegistry.sol

```diff
@@ -16,9 +16,12 @@ contract DAORegistry is InterfaceBasedRegistry {
     /// @notice The ENS subdomain registrar registering the DAO names.
     ENSSubdomainRegistrar private subdomainRegistrar;
 
-    // @notice Thrown if the plugin repository name is empty.
+    /// @notice Thrown if the DAO repository name is empty.
     error EmptyDaoName();
 
+    /// @notice Thrown if the DAO name doesn't match the regex `[0-9a-z\-]`
+    error InvalidDaoName(string name);
```

Comment:
We will be registering an ENS subdomain, not sure if this event name can mislead people

Reply:
it is an error message for `register` where you pass in a name. See line 53

Reply:
changed it to subdomain